### PR TITLE
part1-fixes

### PR DESCRIPTION
**Pull request by vipulgarg1998**

### CODE REVIEW

File: tex/iii.1/_0059_0060.tex [hunk 1]
```diff
@@ -68,6 +68,8 @@ \section{Норма зиску}
 те, що цими засобами виробництва володіють не-робітники, перетворює
 робітників у найманих робітників, а не-робітників — у капіталістів.
 
+\pfbreak
```
Comment (denys-potapov) [inline tex/iii.1/_0059_0060.tex:71]: We can't add \pfbreak at this stage. Pfbreak is kind or chapter break, so if there is one we can replace it with \plainbreak{3}. But not add new ones.

File: tex/iii.1/_0061_0062.tex [hunk 2]
```diff
@@ -12,9 +12,9 @@
 вміщену в ньому оплачену суму праці. Таким чином додаткова
 вартість, звідки б вона не виникала, є надлишок понад увесь
 авансований капітал. Отже, цей надлишок стоїть у такому відношенні
-до всього капіталу, яке виражається дробом \frac{m}{K}, де
+до всього капіталу, яке виражається дробом \fracs{m}{K}, де
```
Comment (denys-potapov) [inline tex/iii.1/_0061_0062.tex:15]: \fracs - typo?

File: tex/iii.1/_0065_0066.tex [hunk 3]
```diff
@@ -94,6 +94,8 @@ \section{Відношення норми зиску до норми
 в силу загальної пересічної норми зиску, про яку
 буде мова в другому відділі.
 
+\plainbreak{4}
```
Comment (denys-potapov) [inline tex/iii.1/_0065_0066.tex:97]: See comment above

File: tex/iii.1/_0079.tex [hunk 4]
```diff
@@ -1,3 +1,4 @@
+\baselineskip
```
Comment (denys-potapov) [inline tex/iii.1/_0079.tex:1]: This what causes strange line break for all pages below it. What was the intention?

File: tex/iii.1/_0083c.tex [hunk 5]
```diff
@@ -30,14 +31,13 @@
 Цей випадок не дає нових точок зору і розв’язується за допомогою
 загальної формули, даної в рубриці: II. $m'$ змінюється.
 
-\pfbreak{}
```
Comment (denys-potapov) [inline tex/iii.1/_0083c.tex:33]: We can't remove the \pfbreak (see above) but can replace it with plainnbreak

File: tex/iii.1/_0086_0087_0088_0089_0090_0091c.tex [hunk 6]
```diff
@@ -157,13 +161,12 @@ \section{Вплив обороту на норму зиску}
...
 обороту змінного капіталу буде:\[
 200c \text{(зношування)} \dplus{} 1000 c \dplus{} 1000 v \dplus{} 1000 m \deq{} 3200,\]
-
 а весь річний продукт при п’яти оборотах:
 
 \begin{center}
 $1000c\text{(зношування)} \dplus{} 5000 c \dplus{} 5000 v \dplus{} 5000 m \deq{} \num{16000}$,
 
-$К \deq{} \num{11000}, m \deq{} 5000, р' \deq{} \frac{5000}{\num{11000}} \deq{} 45\sfrac{5}{11}\%$.
+$К \deq{} \num{11000}, m \deq{} 5000, р' \deq{} \sfrac{5000}{\num{11000}} \deq{} 45\sfrac{5}{11}\%$.
```
Comment (denys-potapov) [inline tex/iii.1/_0086_0087_0088_0089_0090_0091c.tex:169]: generally it's better not to use \sfrac in bloack equations (equations inside {center} or \[\]

**Review by denys-potapov — CHANGES_REQUESTED** [review]
Mostly looks good, but added some clarifications

File: tex/iii.1/_0059_0060.tex (hunk 1 above)
Comment (vipulgarg1998) [reply tex/iii.1/_0059_0060.tex]: Thanks for the explanation. I have fixed this.

File: tex/iii.1/_0061_0062.tex (hunk 2 above)
Comment (vipulgarg1998) [reply tex/iii.1/_0061_0062.tex]: Yeah sorry. I have fixed it.

File: tex/iii.1/_0065_0066.tex (hunk 3 above)
Comment (vipulgarg1998) [reply tex/iii.1/_0065_0066.tex]: Done.

File: tex/iii.1/_0083c.tex (hunk 5 above)
Comment (vipulgarg1998) [reply tex/iii.1/_0083c.tex:33]: Understood. I have reverted back to \pfbreak

File: tex/iii.1/_0086_0087_0088_0089_0090_0091c.tex (hunk 6 above)
Comment (vipulgarg1998) [reply tex/iii.1/_0086_0087_0088_0089_0090_0091c.tex]: Okay, I have reverted them back

File: tex/iii.1/_0131_0132c.tex [hunk 7]
```diff
@@ -100,8 +100,6 @@
 працюючого дрібного селянина, або контролю асоційованих
 виробників.
 
-\pfbreak
```
Comment (denys-potapov) [inline tex/iii.1/_0131_0132c.tex:103]: We can't remove pfbreak if they are already in original file, and can't add them

File: tex/iii.1/_0083c.tex [hunk 8]
```diff
@@ -1,3 +1,4 @@
+\baselineskip
```
Comment (denys-potapov) [inline tex/iii.1/_0083c.tex:1]: And once again, after this command all lines look like they have less linespacing.

**Review by denys-potapov — APPROVED** [review]

**Comment by denys-potapov** [comment]
TODO:
- Revert table position (tex/iii.1/_0106c.tex  
-

**Pull request merged**